requirements/dev.in

```diff
@@ -1,3 +1,3 @@
 -e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]
+-e git+https://github.com/kytos-ng/kytos.git@epic/vlan_range#egg=kytos[dev]
```

Comment:
Comment reminder to revert this pinned branch here once it's ready to land

Reply:
Reverted.